itensor/tensor/algs.cc

```diff
@@ -561,7 +554,15 @@ SVDRefImpl(MatRefc<T> const& M,
     //    orthog(columns(V,nlarge,Mr),2);
     //    }
 
-    orthog(V,2);
+    //orthog(V,2);
+
+
+    QR(V, tempV, R, {"Complete", false, "PositiveDiagonal", true});
+    V &= std::move(tempV);
```

Comment:
Just wanted to point out here I believe this std::move will have no effect, because the &= operator for ITensor::TenRef and related types doesn't currently take advantage of rvalue references and the like. But it's a good idea to try to just move the data of tempV into V, assuming they have the exact same dimensions and strides.

If so, a straightforward way to do this could be to use the V.data() accessor to get a pointer to V's block of memory, and similar for tempV, then do a move on these.

Reply:
Happy to go ahead and merge, though, then work on this as a separate optimization.

Reply:
Ah, I see. I think someone who is more familiar with ITensor::TenRef class than I could probably more effectively and efficiently implement this optimisation, so I think merging now probably makes the most sense.

Reply:
Ok agreed!